.changeset/silent-vans-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+
+---
+
+Added a settings.json file so vscode picks up the correct typescript version
```

Comment:
The changeset seems to be invalid (no package selected)

Reply:
Hmm, technically this doesn't need a changeset since this is a our monorepo change only, it won't be released to the users 🤔